src/Utilities/Utilities.jl

```diff
@@ -435,63 +418,57 @@ function url(repo, file)
         _, path = split(file, root; limit = 2)
         repo = replace(repo, "{commit}", commit)
         repo = replace(repo, "{path}", path)
-        Nullable{Compat.String}(repo)
+        Nullable{String}(repo)
     else
-        Nullable{Compat.String}()
+        Nullable{String}()
     end
 end
 
 url(remote, repo, doc) = url(remote, repo, doc.data[:module], doc.data[:path], linerange(doc))
 
 # Correct file and line info only available from this version onwards.
```

Comment:
Should probably remove this comment as well.

Reply:
Done.